client/implementation-vertx/src/main/java/io/smallrye/graphql/client/vertx/typesafe/VertxTypesafeGraphQLClientProxy.java

```diff
@@ -305,6 +305,7 @@ private JsonObject request(MethodInvocation method) {
         if (clientModel == null) {
             query = queryCache.computeIfAbsent(method.getKey(), key -> new QueryBuilder(method).build());
         } else {
+            clientModel.getOperationMap().keySet().forEach(System.err::println);
```

Comment:
Please remove this line, or use a logger  on debug level

Reply:
Ah, this is logged for each request, so rather remove. This is something that only needs to be logged once, if at all

Reply:
Oh yeah, my bad. I left it while I was debugging.